cryodrgn/fft.py

```diff
@@ -1,44 +1,108 @@
+from itertools import repeat
+import logging
+from multiprocessing.pool import ThreadPool as Pool
 import numpy as np
+import torch
+from torch.fft import fftshift, ifftshift, fft2, fftn, ifftn
 
-try:
-    import cupy as cp  # type: ignore
-except ImportError:
-    cp = np
 
+logger = logging.getLogger(__name__)
 
-def fft2_center(img):
-    pp = np if isinstance(img, np.ndarray) else cp
 
-    return pp.fft.fftshift(
-        pp.fft.fft2(pp.fft.fftshift(img, axes=(-1, -2))), axes=(-1, -2)
+def transform_in_chunks(
```

Comment:
Uhh, I don't know about all this...

I would try to do this stuff on the GPU where it'll be blazing fast. If you're going to do it on CPU, the torch libraries already implement parallelism at a lower level (via OpenMP), you don't want to try to implement parallelism yourself here I think.

Reply:
True. This will go away. Thanks for pointing out.